build.proj

```diff
@@ -33,6 +33,8 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
+
+    <DotnetCliBuildDirectory>$(MSBuildThisFileDirectory)/build_projects/dotnet-cli-build/</DotnetCliBuildDirectory>
```

Comment:
Our directory properties so far haven't been using the trailing slash. Can we keep that consistent? Either we add it to them all or remove from them all. I prefer removing. Less work right now.